sdk/anomalydetector/ai-anomaly-detector-rest/src/anomalyDetectorRest.ts

```diff
@@ -13,15 +13,16 @@ import { AnomalyDetectorRestClient } from "./clientDefinitions";
 export default function createClient(
   Endpoint: string,
   credentials: KeyCredential,
-  options: ClientOptions = {}
+  options: ClientOptions & { apiVersion?: string } = {}
 ): AnomalyDetectorRestClient {
-  const baseUrl = options.baseUrl ?? `${Endpoint}/anomalydetector`;
+  const apiVersion = options.apiVersion ?? "v1.1";
```

Comment:
This is the manual change I had to do:

1. Add apiVersion to clientOptions
2. create the apiVersion variable, if options.apiVersion is not provided then, use the default from swagger.

Reply:
@MaryGao, @qiaozha do you think we can integrate this to the codegen? This is dependent on https://github.com/Azure/autorest/pull/4657

Reply:
Yes I think so, I created an [issue ](https://github.com/Azure/autorest.typescript/issues/1635) to track this